ppx/native/ppx_deriving_json_runtime.ml

```diff
@@ -3,12 +3,31 @@ type t = Yojson.Basic.t
 let to_json t = t
 let of_json t = t
 let to_string t = Yojson.Basic.to_string t
-let of_string s = Yojson.Basic.from_string s
+
+exception Json_error of string
+
+let of_string s =
+  try Yojson.Basic.from_string s
+  with Yojson.Json_error msg -> raise (Json_error msg)
 
 exception Of_json_error of string
 
 let of_json_error msg = raise (Of_json_error msg)
 
+let show_js_type = function
```

Comment:
In this case "js" is an abbreviation of "json" right? As there's a js runtime it's a bit confusing.

Reply:
fixed to be `show_json_type`